src/services/types.ts

```diff
@@ -427,6 +428,8 @@ export interface LanguageServiceHost extends GetEffectiveTypeRootsHost, MinimalR
     getParsedCommandLine?(fileName: string): ParsedCommandLine | undefined;
     /** @internal */ onReleaseParsedCommandLine?(configFileName: string, oldResolvedRef: ResolvedProjectReference | undefined, optionOptions: CompilerOptions): void;
     /** @internal */ getIncompleteCompletionsCache?(): IncompleteCompletionsCache;
+
+    getJSDocParsingKind?(): JSDocParsingKind | undefined;

```

Comment:
After doing more plumbing I feel like I should leave this as a plain property rather than a getter; theoretically someone should only be setting this once.

Reply:
Agree this should be property. Also I think you want this to be passed when creating documentRegistry.  Otherwise there is mismatch between these that you need to handle for releasing the documents.

Reply:
Yep, working on that one now! It's pretty hard to plumb when it's an optional :(